Source/Core/Core/NetworkCaptureLogger.h

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/File.h"
+#include "Common/PcapFile.h"
```

Comment:
Isn't `PCAP` able to be a forward declaration?

Reply:
It doesn't seem to work on non-Windows builds:
> /usr/include/c++/5/bits/unique_ptr.h:74:22: error: invalid application of 'sizeof' to incomplete type 'PCAP'

Reply:
This is because the containing class's (PCAPSSLCaptureLogger) destructor is implicitly defined, which makes it inlineable (inlined destructor + site of incomplete type -> boom). If you default the destructor in the cpp file, it should go away.

Reply:
Done.